roles/kube_prometheus/aggregate_dashboard.ini

```diff
@@ -0,0 +1,8 @@
+[default]
```

Comment:
Instead of requiring a config file, wouldn't it be easier to expect files on the command line and print the result to stdout, so one can run
```
aggregate-dashboard.py ../../vendor/etcd/Documentation/op-guide/grafana.json src/*.json > files/additional_dashboard.yml
```

Reply:
so the configuration lays in the command line. We need a makefile target, or something like this or I'm afraid developers will add their own new dashboard but forget to keep the oldest ones. My belief is that command line is more error prone.

Reply:
In a Makefile you could use `src/*.json` etc without needing any special support in the tool ;-) Anyway, I'll approve, feel free to merge as-is if you feel like it.